perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -83,6 +84,20 @@
                      'Allow swiftclient to access Swift service without \n'
                      'having to verify the SSL certificate')
 
+flags.DEFINE_integer('objects_per_stream', 1000,
```

Comment:
More thoughts: I don't want to prefix the object size flag with "multistream" because, even though it currently only applies to multistream, it clearly could apply to other scenarios, and it would be very natural to do so in the future. I've prefixed the other two with `multistream`.